### PR TITLE
Add MercadoPago payment method.

### PR DESCRIPTION
**Pull request by ghost**
Implements https://github.com/bisq-network/growth/issues/278, adding MercadoPago specifically for the Argentine ARS market.

If / when needed, 6 additional markets ("BR", "CL", "CO", "MX", "PE", "UY") can be added by a simple code change to this payment method.

----

### Screenshots:

mercado1 5

mercado1

mercado1 7

mercado2

mercado3

image

mercado1 8

### CODE REVIEW

**Comment by rodvar** [comment]
Very exciting PR @jmacxx, I'll test/review it soon!!!

File: core/src/main/java/bisq/core/payment/payload/MercadoPagoAccountPayload.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    }
+
+    @Override
+    public String getPaymentDetails() {
+        return Res.get(paymentMethodId) + " - " + getPaymentDetailsForTradePopup().replace("\n", ", ");
+    }
+
+    @Override
+    public String getPaymentDetailsForTradePopup() {
+        return Res.get("payment.mercadoPago.holderId") + ": " + accountHolderId + "\n" +
```
Comment (alvasw) [inline core/src/main/java/bisq/core/payment/payload/MercadoPagoAccountPayload.java:94]: You forgot to add `payment.mercadoPago.holderId` to `core/src/main/resources/i18n/displayStrings.properties`.

**Review by alvasw — CHANGES_REQUESTED** [review]
You forgot to add `payment.mercadoPago.holderId` to `core/src/main/resources/i18n/displayStrings.properties`.

**Review by alejandrogarcia83 — APPROVED** [review]
utACK

**Pull request merged**